src/scrcmd_12.c

```diff
@@ -79,18 +79,18 @@ BOOL ScrCmd_511(ScriptContext *ctx) {
 
```

Comment:
ScrCmd_511 seems to be PalParkGetScore

Reply:
Naming it to `ScrCmd_PalParkScoreGet` for consistency with the command name `pal_park_score_get`